### PR TITLE
[IDB IGNORE] 2021 Halloween Event

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

This is the official PR for Halloween costumes, people who want to contribute to the event must PR their contributions to THIS branch, rather than create a separate fulpstation/fulpstation PR.

More information can be found [on the Forums here](https://forum.fulp.gg/t/halloween-costume-contest/3682).
For instructions on how to participate, [read the Halloween readme here](https://github.com/fulpstation/fulpstation/tree/master/fulp_modules/features/halloween/2021).

### CODE REVIEW

File: fulp_modules/features/halloween/2021/shovel_knight_outfits/propeller_knight.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	desc = "A reflective gold helmet with a makeshift propeller device fastened ontop."
+	icon_state = "propeller_head"
+	body_parts_covered = HEAD
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDEHAIR
+
+///Gloves
+/obj/item/clothing/gloves/costume_2021/propeller_gloves
+	name = "propeller gloves"
+	desc = "A tight yet comfortable pair of gloves."
+	icon_state = "propeller_gloves"
```
Comment (ATH1909) [inline fulp_modules/features/halloween/2021/shovel_knight_outfits/propeller_knight.dm:28]: you could make these (improvised-tier) tackling gloves if you wanted people to try to "fly" with them

File: fulp_modules/features/halloween/2021/shovel_knight_outfits/spectre_knight.dm [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+///Helmet
+/obj/item/clothing/head/costume_2021/spectre_head
+	name = "spectre knight's helmet"
+	desc = "A blood red hood, obscuring reflective golden helmet."
+	icon_state = "spectre_head"
+	body_parts_covered = HEAD
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDEHAIR
+
+/obj/item/storage/box/halloween/edition_21/spectre_knight
+	theme_name = "2021's Spectre Knight"
```
[the remark this answers is not in the document]
Comment (Sheits) [reply fulp_modules/features/halloween/2021/shovel_knight_outfits/spectre_knight.dm:24]: america lol

File: fulp_modules/features/halloween/2021/infinity_clothing/infinity.dm [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		user.dna.species.punchdamagelow += 0.2
+		user.dna.species.punchdamagehigh += 0.1
+
+/obj/item/clothing/gloves/costume_2021/infinity_wristbands/dropped(mob/living/carbon/user)
+	. = ..()
+	if(!ishuman(user))
+		return
+	if(user.get_item_by_slot(ITEM_SLOT_GLOVES) == src)
+		user.dna.species.punchdamagelow -= 0.2
+		user.dna.species.punchdamagehigh -= 0.1
```
[the remark this answers is not in the document]
Comment (JohnFulpWillard) [reply fulp_modules/features/halloween/2021/infinity_clothing/infinity.dm:42]: I don't think it'll cause much of an issue, the probability of that is very low and doing it accidentally is near impossible, and I don't see a problem with people intentionally lowering their punch damage anyways.

File: fulp_modules/features/halloween/2021/shovel_knight_outfits/spectre_knight.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (ATH1909) [reply fulp_modules/features/halloween/2021/shovel_knight_outfits/spectre_knight.dm:24]: it's how it's spelled in the game, from what I can tell

**Comment by JohnFulpWillard** [comment]
image

:raised_eyebrow:

**Pull request merged**